src/main/scala/temple/generate/target/openapi/Response.scala

```diff
@@ -0,0 +1,34 @@
+package temple.generate.target.openapi
+
+import io.circe.syntax._
+
+import io.circe.{Encoder, Json}
+
+import scala.collection.immutable.ListMap
+
+/** https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#responseObject */
+private[openapi] case class Response(
+  description: String,
+  content: Map[String, Response.MediaTypeObject],
+  required: Option[Boolean] = None,
```

Comment:
Kinda annoying that you'd have to wrap it in `Some` to use it, but I see why

Reply:
Can you think of an alternative? Guess I could default to `false`, and only render it if `true`, but you may wish to be more explicit in some cases